### DISCUSSION TITLE
Вызов страницы с кастомным html/js

### DISCUSSION THREAD

**Discussion by Vadimgrig2012** [totumonline/totum-ru-issues-and-discussions#317] [Как это сделать?] [answered]
Добрый день всем. 

Ситуация следующая. Есть файл js, который обрабатывает json и рендерит таблицу в html по определенным условиям и параметрам (средствами тотума такой отчет вывести нельзя, т.к. очень много колонок и они автоматически добавляются по определенным условиям). Через npm были загружены bootstrap (что бы со стилями особо не заморачиваться) и momentjs (обрабатывать даты методом newDate уж слишком ***). Вопрос в том, каким образом возможно скормить всю дату формата json в кастомный js средствами тотума, можно ли каким то образом вызвать данную html страницу через тотум и если все вышеописанное возможно, то как подтянуть библиотеки (возможно через cdn)?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Что-то мы на это не рассчитывали. Если скилл позволяет, можно сделать свой модуль: https://ru.docs.totum.online/additional-interfaces

Довески JS базово заблокированы, бутстрап элементы скорее всего будут пересекаться по css.

Поддерживать этот функционал у нас нет возможности из-за ограниченных ресурсов.
